js/index.js

```diff
@@ -0,0 +1,35 @@
+document.addEventListener('DOMContentLoaded', function() { 
+    const footer = document.createElement('footer')
+// footer.textContent = 'HAUMEA 2024'
+
+
+let today = new Date();
+let thisYear =  today.getFullYear();
+let copyright = document.createElement('span');
+copyright.textContent = "Copyright ";
+
+const fullname = document.createElement('span');
+fullname.textContent = "Anna Bazileeva ";
+
+let unicode = document.createElement('span');
+unicode.textContent = " © ";
+
+footer.appendChild(copyright);
+footer.appendChild(fullname); 
+footer.appendChild(unicode)
+footer.append(` ${thisYear}`)
```

Comment:
Great job creating the footer, Anna! I can see you understand the difference between `appendChild` and `append` methods.

I just have a couple pointers on this:
1. The assignment said to use a paragraph element (`p`).
2. You don't need an element for each part of the text, they're only one thing. Instead of creating a `span` for each part of the text, create a single `string` concatenating them and set that as the content of the paragraph.

Here's an approach that uses both the `append` and the `appendChild` methods:

```suggestion
let copyright = document.createElement('p');
copyright.append(`Copyright Anna Bazileeva © ${thisYear}`);

footer.appendChild(copyright);
```